src/main/kotlin/pt/up/fe/ni/website/backend/util/FileUploader.kt

```diff
@@ -0,0 +1,47 @@
+package pt.up.fe.ni.website.backend.util
```

Comment:
I apologize for my aggression but I firmly believe the utils package can be the most common anti-pattern there is (https://www.yanglinzhao.com/posts/utils-antipattern/). `util` gives no information about what we can find inside it, tends to grow way too much, and most code you usually find in there could be separated in meaningful packages.

In this case, FileUploader seems like a clear service for uploading files so I think you should move these 3 classes there. I would also put them in separate files under `services/upload` for example. Everything else should work the same way w/ the config

Reply:
I redone the repo structure according to your suggestion of using utils with extensions and annotations